auth_ldap_attribute_sync/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
for new modules, use [split readme files](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)

Reply:
Done